python/lsst/meas/pz/estimate_pz_task.py

```diff
@@ -425,8 +428,8 @@ def _get_mags_and_errs(
         mag_dict = {}
         # loop over bands, make mags and mag errors and fill dict
         for band in flux_names.keys():
-            fluxVals = fluxes[flux_names[band]]
-            fluxErrVals = fluxes[flux_err_names[band]]
+            fluxVals = fluxes[flux_names[band]].data
```

Comment:
This is to get rid of the astropy unit from an astropy table column, right? I think you can also do `np.array(fluxes...)` and that will generally work, though I'm not sure if it avoids the copy.

Reply:
Yes, it is to pass a dict of numpy arrays to RAIL.   Whatever works.